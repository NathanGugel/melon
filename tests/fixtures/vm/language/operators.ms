io.print(2^32);
io.print(2 % 2);
io.print(3 % 2);
io.print((1 + 1)^(10 * 3 + 2));
io.print(4.0^0.5);