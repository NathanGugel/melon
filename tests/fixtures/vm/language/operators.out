4294967296
0
1
4294967296
2.000000000
